### ISSUE TITLE
Add alerts for unconnected hardware

### ISSUE THREAD

**Issue by allengregoryiv** [Spectrum3847/2025-SpectrumBeta#25]
We should error check the CAN calls and when we fail, retry them, and if we fail the full loop display an alert that the device is disconnected.

This should happen for motors in the Mechanism class.

Likely need to add something to Swerve as well.

Should happen for CAN sensors, as well as Limelights and other devices.

**Comment by cycIes** [comment]
@allengregoryiv 
Is this what you mean?

`protected void setMotorPosition(DoubleSupplier rotations) {
        if (isAttached()) {
            StatusCode response = motor.setPosition(rotations.getAsDouble());
            if (response.isOK()) {
                motor.setPosition(rotations.getAsDouble());
            } else {
                response = motor.setPosition(rotations.getAsDouble());
                if (response.isOK()) {
                    motor.setPosition(rotations.getAsDouble());
                } else {
                    motorDisconnectedAlert.set(true);
                }
            }
        }
    }`

**Comment by allengregoryiv** [comment]
Yes I think that looks pretty good, we should test that it works on a robot.

**Comment by allengregoryiv** [comment]
One of the things I think we should start doing is using fewer indents. - https://www.youtube.com/watch?v=CFRhGnuXG-4&list=PLP7PdOqmqKAEOb6R-Byr7gySYHgLoYvPY&index=14

So instead of doing

```
if (isAttached()) {
            ALL THE CODE
        }
```

        
 We do 

```
 if(!isAttached(){
     return;
 }
 
 ALL THE CODE
```

**Comment by allengregoryiv** [comment]
I also think we are double-setting the motor.

So we can remove the setPosition after we get an Ok response since we already set the position before.

**Comment by allengregoryiv** [comment]
We should also make Alerts that show if a motor is licensed to CTRE, so we know we need to license it.
